src/GitHub.Exports/Primitives/UriString.cs

```diff
@@ -141,11 +143,15 @@ public Uri ToRepositoryUrl()
                 ? url.Scheme
                 : Uri.UriSchemeHttps;
 
+            var nameWithOwner = owner == null || NameWithOwner == null ?
+                NameWithOwner :
+                string.Format(CultureInfo.InvariantCulture, "{0}/{1}", owner, RepositoryName);
+
```

Comment:
Nit: Given `owner = null` and `NameWithOwner` != null, won't this mean that `nameWithOwner` will be `/bar` instead of `bar`? I know it probably doesn't make a difference if the `Path` in UriBuilder` starts with a `/`, it will probably get ignored, but wouldn't this be more straightforward as:

```
var nameWithOwner = owner != null ? string.Format(CultureInfo.InvariantCulture, "{0}/{1}", owner, RepositoryName) : NameWithOwner;
```

Also, if `RepositoryName` is null (it can be), should this probably return null even if `owner` is passed in, for consistency?